packages/default-reporter/src/reporterForClient/reportLifecycleScripts.ts

```diff
@@ -264,3 +266,18 @@ function formatLine (maxWidth: number, logObj: LifecycleLog) {
 function cutLine (line: string, maxLength: number) {
   return stripAnsi(line).substr(0, maxLength)
 }
+
+function bufferByDepPath (source: Rx.Observable<LifecycleLog>) {
+  return source.pipe(
+    groupBy(data => data.depPath),
```

Comment:
should I use `depPath` or `wd` property ?

Reply:
I guess it doesn't matter. They will both be unique and will work for grouping